custom/_legacy/mvp/reports/va.py

```diff
@@ -0,0 +1,302 @@
+import datetime
+from couchdbkit import ResourceNotFound
+import dateutil
+from django.utils.safestring import mark_safe
+import logging
+import numpy
+import pytz
+from corehq.apps.indicators.models import DynamicIndicatorDefinition, CombinedCouchViewIndicatorDefinition
+from dimagi.utils.decorators.memoized import memoized
+from mvp.models import MVP
+from mvp.reports import MVPIndicatorReport
+
+
+class VerbalAutopsyReport(MVPIndicatorReport):
+    slug = "verbal_autopsy"
+    name = "Verbal Autopsy Report"
+    report_template_path = "mvp/reports/verbal_autopsy.html"
+    flush_layout = True
+    hide_filters = True
+    fields = ['corehq.apps.reports.filters.users.UserTypeFilter',
+              'corehq.apps.reports.filters.select.GroupFilter']
+    emailable = True
+
+    @property
+    def use_new_db(self):
```

Comment:
can also do `use_new_db = True` since no computation has to happen...but this is ok and won't block a merge